docs/about/digital.md

```diff
@@ -102,21 +102,21 @@ To facilitate reuse and analysis of the Project's edited and encoded text of Ms.
 Research and Scholarly Essays
 ---
 
-All [essays](/#essays) are edited and maintained in the Project's Google Drive in order to best facilitate the collaborative process. Each essay in the Drive has uniform organization and style to ensure consistency and allow for conversion to HTML and integration into this Edition. The Research and Scholarly Essays are supplemented with additional information (including Project-assigned themes, associated entries, and citation information) in a spreadsheet table and exported in CSV format for integration into the Edition.
+All [essays](/#essays) are edited and maintained in the Project's Google Drive in order to best facilitate the collaborative process. Each essay in the Drive has uniform organization and style to ensure consistency and allow for conversion to HTML and integration into the Edition. The Research and Scholarly Essays are supplemented with additional information (including Project-assigned themes, associated entries, and citation information) in a spreadsheet table and exported in CSV format for integration into the Edition.
 
 For persistent access, each essay has been assigned a Digital Object Identifier (DOI) and will also be archived in Columbia University [Academic Commons](https://academiccommons.columbia.edu/) and downloadable as standalone publications.
 
 Static Site Generation
 ------
 
-Most websites are typically created using a content management system (CMS) in which the site's content is kept in a database and rendered into HTML when requested. This approach often requires the employment of proprietary software as well as a large number of resources, institutional support, or constant maintenance. Static site generation, unlike CMS or database-driven projects, allows websites to be created with significantly fewer dependencies, reduces maintenance overhead, and minimizes the risk of technological obsolescence. Content is already in its final form (as static pages are created beforehand) and can be sent as-is to the user.
+Most websites are typically created using a content management system (CMS) in which the site's content is kept in a database and rendered into HTML when requested. This approach often requires the employment of proprietary software as well as a large number of resources, institutional support, or constant maintenance. Static site generation, unlike CMS or database-driven projects, allows websites to be created with significantly fewer dependencies, reducing maintenance overhead, and minimizing the risk of technological obsolescence. Content is already in its final form (as static pages are created beforehand) and can be sent as-is to the user.
```

Comment:
I think that we should keep the original because it is more technically precise. Static site generation does these three things (allows, reduces, minimizes) rather than the "fewer dependencies" causing the last two things

Reply:
change to:
> static site generation, unlike CMS or database-driven projects, allows websites to be created with significantly fewer dependencies, and reduces maintenance overhead, as well as minimizes the risk of technological obsolescence. Content is already in its final form (as static pages are created beforehand) and can be sent as-is to the user.